posthog/hogql/ast.py

```diff
@@ -506,7 +514,7 @@ class FieldType(Type):
     table_type: TableOrSelectType
 
     def resolve_database_field(self, context: HogQLContext) -> Optional[FieldOrTable]:
-        if isinstance(self.table_type, BaseTableType):
+        if isinstance(self.table_type, BaseTableType) or isinstance(self.table_type, SelectViewType):
```

Comment:
Unsure on this case - a `SelectViewType` is not a `Table` in hogql land (hence it not being covered by `BaseTableType`)

Reply:
Yeah, I understand. We can solve this in a different way, but my problem is the following, this errors out when I'm trying to add a pretty join to a view

<img width="733" height="507" alt="image" src="https://github.com/user-attachments/assets/124535bd-4af2-4089-a893-3906769d34e9" />

It does work in the end, but it's in red